QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,123 @@
+
+### API Stability Within a Released ROS Distribution [1.iv]/[1.vi]
+
+`foonathan_memory_vendor` will not break public API within a released ROS distribution, i.e. no major releases once the ROS distribution is released.
+
+CMake files that are not in the cmake directory are not installed, and are considered private.
+
+### ABI Stability Within a Released ROS Distribution [1.v]/[1.vi]
+
+`foonathan_memory_vendor` does not contain any c or c++ code, therefore changes will not affect ABI stability.
+
+## Change Control Process [2]
+
+`foonathan_memory_vendor` follows the recommended guidelines for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#package-requirements).
+
+### Change Requests [2.i]
+
+This package requires that all changes occur through a pull request.
+
+### Contributor Origin [2.ii]
+
+This package uses DCO as its confirmation of contributor origin policy. More information can be found in [CONTRIBUTING](CONTRIBUTING.md).
+
+### Peer Review Policy [2.iii]
+
+Following the recommended guidelines for ROS Core packages, all pull requests must have at least 1 peer review.
+
+### Continuous Integration [2.iv]
+
+All pull request must pass CI on all [tier 1 platforms](https://www.ros.org/reps/rep-2000.html#support-tiers)
+
+### Documentation Policy [2.v]
+
+All pull requests must resolve related documentation changes before merging.
+
+## Documentation [3]
+
+### Feature Documentation [3.i]
+
+`foonathan_memory_vendor` does not currently document its features.
+
+### Public API Documentation [3.ii]
+
+`foonathan_memory_vendor` has embedded API documentation, and is available in its source files.
+There is documentation for all of the public API, and new additions to the public API require documentation before being added.
+
+### License [3.iii]
+
+The license for `foonathan_memory_vendor` is Apache 2.0, and a summary is in each source file, the type is declared in the [package.xml](package.xml) manifest file, and a full copy of the license is in the [LICENSE](../LICENSE) file.
+
+There is an automated test which runs a linter that ensures each file has a license statement.
+
+Most recent test results can be found [here](https://ci.ros2.org/view/nightly/job/nightly_linux_release/lastBuild/testReport/foonathan_memory_vendor/copyright/).
+
+### Copyright Statements [3.iv]
+
+The copyright holders each provide a statement of copyright in each source code file in `foonathan_memory_vendor`.
+
+There is an automated test which runs a linter that ensures each file has at least one copyright statement.
+
+Results of the copyright tests can be found [here](https://ci.ros2.org/view/nightly/job/nightly_linux_release/lastBuild/testReport/foonathan_memory_vendor/copyright/).
+
+## Testing [4]
+
+`foonathan_memory_vendor` is a package providing solely CMake files and therefore does not require tests and has no coverage or performance requirements.
```

Comment:
I think it would be great adding a reference to the points skipped, it's weird jumping from [4] to [4.v], something like:

```suggestion
`foonathan_memory_vendor` is a package providing solely CMake files and therefore does not require tests and has no coverage or performance requirements. [4.i], [4.ii], [4.iii] & [4.iv]
```

Reply:
Done.